src/test/java/net/sf/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -128,7 +128,7 @@ public void findMissingLocalizationKeys() throws IOException {
                 missingKeys.parallelStream()
                         .map(key -> String.format("%s=%s", key.getKey(), key.getKey()))
                         .collect(Collectors.toList()),
-                Collections.EMPTY_LIST, missingKeys);
+                Collections.<LocalizationEntry>emptyList(), missingKeys);
```

Comment:
Just for the future, you could have just taken the Collections.emptyList() without specifying the type explicit.